pkg/apis/serving/v1alpha1/kfservice_validation.go

```diff
@@ -89,12 +87,15 @@ func validateReplicas(minReplicas int, maxReplicas int) error {
 	return nil
 }
 
-func validateDefaultSpec(defaultSpec ModelSpec) error {
-	if err := validateOneModelSpec(defaultSpec); err != nil {
+func validateSpec(spec ModelSpec) error {
```

Comment:
I think this should be validateModelSpec since spec typically refers to the top level spec.